drivers/volume/portworx/portworx.go

```diff
@@ -164,6 +175,16 @@ type portworxGrpcConnection struct {
 	endpoint    string
 }
 
+func isTLSEnabled() bool {
+	tls := strings.ToLower(os.Getenv(pxEnableTLS))
+	if len(tls) != 0 ||
+		tls == "true" || tls == "yes" ||
```

Comment:
You don't need to check for all these combinations. Just use `strconv.ParseBool()`
https://golang.org/pkg/strconv/#ParseBool

Reply:
oh wow, i didn't know about that API. Thanks!